<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Titration</class>
 <widget class="QWidget" name="Titration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table_titration">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>65</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>15</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Initial</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Flag</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Buret</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Flag</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>L</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>0.001</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>H</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>0.005</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>-0.1</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="lbl_points">
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="dsb_V0">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbl_Vf">
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_Vi">
       <property name="text">
        <string>Initial</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dsb_Vf">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>11.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="sb_NPoints">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Error</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="dsb_Verror">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="value">
        <double>0.003000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
